website/docs/docs/deploy/model-notifications.md

```diff
@@ -42,22 +44,26 @@ groups:
   - name: finance
     description: "Models related to the finance department"
     owner:
-      # 'name' or 'email' is required
+      # Email is required to receive model-level notifications, additional properties are also allowed.
       name: "Finance Team"
       email: finance@dbtlabs.com
-      slack: finance-data
+      favorite_food: donuts
 
   - name: marketing
     description: "Models related to the marketing department"
     owner:
       name: "Marketing Team"
       email: marketing@dbtlabs.com
-      slack: marketing-data
+      favorite_food: jaffles
 ```
 
-## Set up models
+</File>
+
+## Attach groups to models
+
+Attach groups to models as you would any other config, in either the `dbt_project.yml` or `<whatever>.yml` files. For example: 
 
-Set up your model configuration in either the `dbt_project.yml` or `groups.yml` file; doing this automatically sets up notifications for tests, too. For example: 
+<File name='models/marts.yml'>
```

Comment:
The rest of this is purely additive and you can rework as you see fit

Reply:
thanks for adding the files tag! 🙏